pkg/apis/workflow/v1alpha1/info.go

```diff
@@ -8,6 +8,8 @@ type Link struct {
 	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
 	// "workflow", "pod", "pod-logs", "event-source-logs", "sensor-logs", "workflow-list" or "chat"
 	Scope string `json:"scope" protobuf:"bytes,2,opt,name=scope"`
+	// Target attribute specifies where a linked document will be opened when a user clicks on a link. E.g. "_blank", "_self". If the target is _blank, it will open in a new tab.
+	Target string `json:"target" protobuf:"bytes,4,opt,name=target"`
 	// The URL. Can contain "${metadata.namespace}", "${metadata.name}", "${status.startedAt}", "${status.finishedAt}" or any other element in workflow yaml, e.g. "${workflow.metadata.annotations.userDefinedKey}"
 	URL string `json:"url" protobuf:"bytes,3,opt,name=url"`
```

Comment:
The order is incorrect
```suggestion
	// The URL. Can contain "${metadata.namespace}", "${metadata.name}", "${status.startedAt}", "${status.finishedAt}" or any other element in workflow yaml, e.g. "${workflow.metadata.annotations.userDefinedKey}"
	URL string `json:"url" protobuf:"bytes,3,opt,name=url"`
	// Target attribute specifies where a linked document will be opened when a user clicks on a link. E.g. "_blank", "_self". If the target is _blank, it will open in a new tab.
	Target string `json:"target" protobuf:"bytes,4,opt,name=target"`
```

Reply:
OK, Modified.